src/main/java/model/LottoNumber.java

```diff
@@ -0,0 +1,56 @@
+package model;
+
+import exception.InvalidRangeLottoNumberException;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import util.NumberFormatStringParser;
+
+public class LottoNumber {
+    private static final int MINIMUM_LOTTO_NUMBER = 1;
+    private static final int MAXIMUM_LOTTO_NUMBER = 45;
+    private final int lottoNumber;
+
+    public LottoNumber(int number) {
+        if (isInvalidRange(number)) {
+            throw new InvalidRangeLottoNumberException();
+        }
+        this.lottoNumber = number;
+    }
+
+    public static LottoNumber parse(String text) {
+        int number = NumberFormatStringParser.parse(text);
+        return new LottoNumber(number);
+    }
```

Comment:
`Integer.parseInt`를 그대로 사용하면 안되나요?

```suggestion
    public static LottoNumber parse(String number) {
        return new LottoNumber(Integer.parseInt(number));
    }
```